src/lib.rs

```diff
@@ -2545,4 +2564,41 @@ mod tests {
         assert!(rtshark.pid().is_none());
         tmp_dir.close().expect("Error deleting fifo dir");
     }
-}
\ No newline at end of file
+
+    #[test]
+    fn test_reassembled_tcp() {
+        let pcap = include_bytes!("tcp_fragmentation.pcap");
+
+        // create temp dir and copy pcap in it
+        let tmp_dir = tempdir::TempDir::new("test_pcap").unwrap();
+        let pcap_path = tmp_dir.path().join("file.pcap");
+        let mut output = std::fs::File::create(&pcap_path).expect("unable to open file");
+        output.write_all(pcap).expect("unable to write pcap");
+        output.flush().expect("unable to flush");
```

Comment:
I copied this from the other tests, but I'm not actually sure why all the test needs to read from a copy of the pcap. Is it necessary?